src/screens/transactions-history/list-transactions-history/list-transactions-history.tsx

```diff
@@ -74,7 +74,19 @@ export class TransactionsHistoryListComponent extends React.Component<
         return (
             <TouchableOpacity
                 key={tx.id}
-                style={styles.transactionListItem}
+                style={[
+                    styles.transactionListItem,
+                    {
+                        backgroundColor:
+                            tx.status === TransactionStatus.PENDING
+                                ? theme.colors.cardBackground
+                                : theme.colors.appBackground,
+                        paddingVertical:
+                            tx.status === TransactionStatus.PENDING
+                                ? BASE_DIMENSION * 2
+                                : BASE_DIMENSION
+                    }
+                ]}
```

Comment:
you can move this in styles
stylePending { backgroundColor: theme.colors.cardBackground, paddingVertical: BASE_DIMENSION*2}
styleOther { backgroundColor: appBackground, paddingVertical: BASE_DIMENSION}